### PR TITLE
Update README for 5.7

### PR DESCRIPTION
**Pull request by tom-on-the-internet**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,20 +4,21 @@
...
 
-| Laravel version | Laravel-Markdown version                                            |
-| --------------- | ------------------------------------------------------------------- |
-| 5.6             | [3.0](https://github.com/andreasindal/laravel-markdown/tree/master) |
-| 5.5             | [2.0](https://github.com/andreasindal/laravel-markdown/tree/2.0)    |
-| 5.3, 5.4        | [1.1](https://github.com/andreasindal/laravel-markdown/tree/1.1)    |
-| 5.2             | [1.0](https://github.com/andreasindal/laravel-markdown/tree/1.0)    |
+| Laravel version | Laravel-Markdown version                                              |
+| --------------- | --------------------------------------------------------------------  |
+| 5.7             | [3.0.1](https://github.com/andreasindal/laravel-markdown/tree/master) |
```
Comment (juliano-barros) [inline README.md]:
```suggestion
| 5.7             | [3.0.1](https://github.com/andreasindal/laravel-markdown/tree/3.0.1) |
```

**Review by juliano-barros — CHANGES_REQUESTED** [review]
Good idea, It was hard for me to find out there was a new version for the new version of Laravel. Although everything is fine I'd like to suggest a slight change.

**Comment by alsterholm** [comment]
Hi! Thanks for your submission. I'm sorry that it has taken me so long to respond. If you make the change according to @juliano-barros suggestion I'll merge it tonight. Thank you!

**Comment by tom-on-the-internet** [comment]
@andreasindal Sorry, I didn't see you had replied. Changes merged in.

**Comment by alsterholm** [comment]
Awesome, thank you for your contribution! 🎉✨

**Pull request merged**